systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java

```diff
@@ -435,20 +435,22 @@ protected boolean isBrokered(AddressSpace addressSpace) {
         return addressSpace.getType().equals(AddressSpaceType.BROKERED);
     }
 
-    protected void assertCanConnect(AddressSpace addressSpace, UserCredentials credentials, List<Destination> destinations) throws Exception {
+    protected boolean assertCanConnect(AddressSpace addressSpace, UserCredentials credentials, List<Destination> destinations) throws Exception {
         for (Destination destination : destinations) {
             String message = String.format("Client failed, cannot connect to %s under user %s", destination.getType(), credentials);
             AddressType addressType = AddressType.getEnum(destination.getType());
             assertTrue(canConnectWithAmqpAddress(addressSpace, credentials, addressType, destination.getAddress(), true), message);
         }
+        return true;
     }
 
-    protected void assertCannotConnect(AddressSpace addressSpace, UserCredentials credentials, List<Destination> destinations) throws Exception {
+    protected boolean assertCannotConnect(AddressSpace addressSpace, UserCredentials credentials, List<Destination> destinations) throws Exception {
```

Comment:
Whats the point in returning anything? An assert typically never returns anything, either it fails or it succeeds

Reply:
ah, you're right tha's my bad, I forgot it there.